prow/getting_started_kind_minikube.md

```diff
@@ -0,0 +1,109 @@
+```
+### minikube preparation
+Alternatively, of you use minikube:
+```
+$ minikube start
+```
+Due to a deticated `minikube ip` command, you do not need to inspect IP address assigned to minikube. Just run the following command:
+```
+$ smee -u https://smee.io/YourWebhookHashHere --target $(minikube ip):30003/hook
+```
+Apply prow CRDs using:
+```
+$ kubectl apply --server-side=true -f https://raw.githubusercontent.com/kubernetes/test-infra/master/config/prow/cluster/prowjob-crd/prowjob_customresourcedefinition.yaml
+```
+
+### Update the sample manifest
+The sample manifest you need to use is [starter-s3-kind.yaml](config/prow/cluster/starter/starter-s3-kind.yaml).
+There are several fields that has to be configured before applying the manifest:
+
+* The github app cert by replacing the `<<insert-downloaded-cert-here>>` string
+* The github app id by replacing the `<<insert-the-app-id-here>>` string
+* The hmac token by replacing the `<< insert-hmac-token-here >>` string
+* The repository by replacing the `<< your_github_repo >>` string (format: `user/repo`)
+* The IP address by replacing the `<< your-minikube/kind IP >>` string
+* The minio's root account by replacing the `<<CHANGE_ME_MINIO_ROOT_USER>>` string in 3 places
+* The minio's secret by replacing the `<<CHANGE_ME_MINIO_ROOT_USER>>` string in 3 places
+
+After filling all the fields, you need to apply the manifest. It can be done using:
+```
+$ kubectl apply -f starter-s3-kind.yaml 
+```
+A few minutes later, all the components should be running (some restarts are possible due to a slow minio init):
+```
+$ kubectl -- get pods -n prow
+NAME                                       READY   STATUS    RESTARTS       AGE
+crier-5db74d86c4-zkf4h                     1/1     Running   0              3m38s
+deck-85d8978769-bvbvz                      1/1     Running   0              3m39s
+ghproxy-5c5df8f459-kgxp7                   1/1     Running   0              3m38s
+hook-7c9bf9b48b-vpkx7                      1/1     Running   0              3m38s
+horologium-696596964b-gjwkn                1/1     Running   0              3m38s
+minio-69fb9fff98-j59dq                     1/1     Running   0              3m37s
+prow-controller-manager-7d8988546c-tc4dz   1/1     Running   0              3m38s
+sinker-69f69c4f85-ssl8h                    1/1     Running   0              3m38s
+statusreconciler-64f5c88b7-hfnxb           1/1     Running   0              3m38s
+tide-74f555b5b5-65tsq                      1/1     Running   5 (115s ago)   3m38s
+```
+
+### Static ports for hook, deck and minio console
+You can access ``hook`` using port ``30001``, ``deck`` UI is listening on port ``30002`` and ``minio`` UI on port ``30003``. Use ``minikube ip`` or the method described previously for kind to get the IP address. Taking `192.168.49.1` as an example IP address, go to the web browser and type `192.168.49.1:30002` to access `deck`.
```

Comment:
I am using a load balancer for this as it assures a static external IP port. I am using a range from 30000 as it is recommended for minikube.